refinery/data_set_manager/views.py

```diff
@@ -1018,3 +1018,41 @@ def post(self, request):
                 import_file.delay(file_store_item.uuid)
 
         return HttpResponse()  # 200
+
+
+class RemoveFileFromNodeView(View):
+    """Remove file from a Node instance"""
+    def post(self, request):
+        try:
```

Comment:
maybe worth setting a variable node_uuid = request.POST['node_uuid'].  I think it would improve readability

Reply:
I don't think introducing an extra variable that's used only once would increase readability.